pkg/gui/patch_building_panel.go

```diff
@@ -19,7 +19,8 @@ func (gui *Gui) refreshPatchBuildingPanel(selectedLineIdx int) error {
 	}
 
 	to := gui.State.Contexts.CommitFiles.CommitFileTreeViewModel.GetRefName()
-	from, reverse := gui.State.Modes.Diffing.GetFromAndReverseArgsForDiff(to)
+	parent := gui.State.Contexts.CommitFiles.CommitFileTreeViewModel.GetParentRefName()
```

Comment:
Looks like we always want to store the two refs at once, and to obtain the two refs at once. If so, could we have a get and set method that takes/returns both refs? E.g. GetRefNames() which returns the ref and parent ref.